GithubCommentReader/index.js

```diff
@@ -242,7 +253,7 @@ const commands = (/** @type {Map<RegExp, CommentAction>} */(new Map()))
             target_branch: pr.head.ref
         })};
     })))
-    .set(/user test this(?! slower)/, action(async request => await makeNewBuildWithComments(request, "parallelized community code test suite", 33, async p => {
+    .set(/user test this(?! slower|inline)/, action(async request => await makeNewBuildWithComments(request, "parallelized community code test suite", 33, async p => {

```

Comment:
```suggestion
    .set(/user test this(?! slower| inline)/, action(async request => await makeNewBuildWithComments(request, "parallelized community code test suite", 33, async p => {

```
The whitespace in the regex has to be in both branches of the `|`.

Reply:
You're right. Fixed.